packages/features/ee/api-keys/components/ApiKeyDialogForm.tsx

```diff
@@ -62,7 +62,7 @@ export default function ApiKeyDialogForm({
           </div>
           <div>
             <div className="flex">
-              <code className="bg-subtle text-default mb-2 w-full truncate rounded-md rounded-r-none py-[6px] pl-2 pr-2 align-middle font-mono">
+              <code className="bg-subtle text-default w-full truncate rounded-md rounded-r-none py-[6px] pl-2 pr-2 align-middle font-mono">
```

Comment:
The "mb-2" className was removed to fix this wrong input size
![245198211-6b727cf9-5bc0-410e-bcf7-865918194c89](https://github.com/calcom/cal.com/assets/121884634/72bdd5ba-cb5b-4d4c-ad2e-c76a7fc3b7aa)